hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/RegionServerServices.java

```diff
@@ -93,11 +93,14 @@ class PostOpenDeployContext {
     private final HRegion region;
     private final long openProcId;
     private final long masterSystemTime;
+    private final long masterStartCode;
```

Comment:
Better add some comments or give it a more clear name so developers will know that this is the start code of the master which scheduled this procedure, for fencing.